vaapi/vaapidisplay_unittest.cpp

```diff
@@ -134,19 +121,6 @@ VAAPI_DISPLAY_TEST(CreateInvalidHandleDRM) {
     EXPECT_FALSE(display.get());
 }
 
-VAAPI_DISPLAY_TEST(CreateInvalidHandleVA) {
-    NativeDisplay native = {0, NATIVE_DISPLAY_VA};
-    DisplayPtr display = VaapiDisplay::create(native);
-
-    EXPECT_FALSE(display.get());
-
-    native.handle = -1;
-    native.type = NATIVE_DISPLAY_VA;
-    display = VaapiDisplay::create(native);
-
-    EXPECT_FALSE(display.get());
-}
-
```

Comment:
There is nothing wrong with testing "special" values, especially for the "negative" path.  No matter how silly it may seem, there will always be someone who tries to do it.  We don't always want to test only the "good" (or "happy") path.  You never know when someone will come along and "accidentally" mess up the internal logic that handles 0 and -1 cases.  This test will catch that if no one else does.

IMO, any test is better than "no" test, no matter how trivial or inane it might seem.

BTW, if I use -2 or 1, I get a valid DisplayPtr... that is probably not good.  Seems to me that our intention is that a valid DisplayPtr means it's okay to use it.  In these cases, it is not okay.  Seems to me that we should be checking vaDisplayIsValid during the create function... and write separate DEATH tests to verify the expectation of such.